save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -48,6 +48,18 @@ class VulnerabilityController(
     private val tagService: TagService,
     private val cosvService: CosvService,
 ) {
+    @GetMapping("count-approved")
+    @Operation(
+        method = "Get",
+        summary = "Get number of all approved vulnerabilities (including AUTO_APPROVED)",
+        description = "Count all approved vulnerabilities",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully fetched the number of approved vulnerabilities")
+    fun getNumberOfApprovedVulnerabilities(): Mono<Int> = blockingToMono {
+        vulnerabilityService.countByStatus(VulnerabilityStatus.APPROVED) +
```

Comment:
may be we can change `countByStatus` (in service) such that it will take list of statuses, or smth like this?

Reply:
removed